ts/utils/payment.ts

```diff
@@ -440,3 +446,20 @@ export const getPickPaymentMethodDescription = (
       })
     : fromNullable(paymentMethod.info.holder).getOrElse(defaultHolder);
 };
+
+export const isDuplicatedPayment = (
+  error: Option<
+    PayloadForAction<
+      | typeof paymentVerifica["failure"]
+      | typeof paymentAttiva["failure"]
+      | typeof paymentIdPolling["failure"]
+    >
+  >
+) =>
+  error
+    .map(
```

Comment:
You can replace the `map` + `getOrElse` with an `exists`:

```ts
error.exists(
  detail =>
    detail === "PAA_PAGAMENTO_DUPLICATO" ||
    detail === "PPT_PAGAMENTO_DUPLICATO"
);
```

I think it's a bit more readable, what do you think? 🙂

Reply:
You never stop learning :) 820c267c2463076e026e5c02fa7ae9532c8025ac